### DISCUSSION TITLE
VoiScribe: An iOS App for On-Device Speech Recognition

### DISCUSSION THREAD

**Discussion by jumon** [ggml-org/whisper.cpp#562] [Show and tell]
Hello!

Thank you for developing such a great project! We are excited to share with you VoiScribe, our iOS application for on-device automatic speech recognition, which is based on whisper.cpp.

Key features:
- Secure offline speech recognition with Whisper
- Intuitive and sophisticated user interface
- Easy transcription editing
- Compatibility with iPhones and iPads (the Mac version is currently in progress)
- Audio file and transcription export
- Upcoming iCloud sync

We are currently looking for beta testers for VoiScribe before its public release. If you are interested, please click on the following link https://testflight.apple.com/join/LhaENwM5 to install the beta version of the app.

Thank you for your support, and we look forward to hearing from you soon!

<p float="left">
  <img width="240" src="https://user-images.githubusercontent.com/34873661/222777320-875ea2a8-91d8-401e-9df9-f26335a474ba.jpg">
  <img width="240" src="https://user-images.githubusercontent.com/34873661/222777658-8c6c41d3-da0a-410c-a942-c12dd5ffc8d5.jpg">
  <img width="240" src="https://user-images.githubusercontent.com/34873661/222776937-e08763a9-f8d9-4090-aa91-59eb0ce10dbf.png">
</p>

**Comment by gardner** [comment]
Is the source code available?

**Reply by jumon** [reply]
Thank you for your interest! Currently, the source code is not available, but we might make it open source in the future. If we do decide to make it available, we will inform you in this thread.

**Reply by gardner** [reply]
So, just to clarify:

* You took an open source project
* Made a closed source thing with it
* Are spamvertising it on the open source repo

Not cool.

**Reply by danglingptr0x0** [reply]
> Thank you for your interest! Currently, the source code is not available, but we might make it open source in the future. If we do decide to make it available, we will inform you in this thread.

reported

**Comment by boringpapa** [comment]
Will you update this to detect other languages?
